docs/common_errors.md

```diff
@@ -8,40 +8,38 @@ progress.)
 
 ## I'm getting kind errors when implementing a `Carrier` instance!
 
-Given a `Teletype` data type:
+Given an effect datatype that doesn’t use the `m` parameter:
```

Comment:
`Teletype` now uses `m`, so the only effects falling into this category are ones which have no continuations whatsoever, e.g. `Fail`.

Reply:
Honestly now that almost every effect uses its `m`, we could probably nuke this whole documentation.

Reply:
I had considered that, but `Fail` doesn’t, and it’s conceivable that you might write your own fail-alike effect, e.g. #154.